task-list/app/views/tasks/new.html.erb

```diff
@@ -1,13 +1,4 @@
-<%= form_for @task do |f| %>
-  <fieldset>
-    <legend> Add new Task </legend>
-      <p id="name"><%= f.label :name %>:
-      <%= f.text_field :name %> </p>
 
-      <p id="description"> <%= f.label :description %>
-      <%= f.text_field :description %></p>
-      <%= f.submit class: "submit"%>
-  </fieldset>
 
-
-  <% end %>
+<!-- <% render partial: 'form', locals: {method_type: :patch, button_text: "New Task"} %> -->
```

Comment:
Even though this is inside of an HTML comment, the ERB tag on this line will still be run!

However, because it's a code tag and not a value tag, the Ruby code will run and then the result will be ignored. So your final HTML output won't have anything but an empty HTML comment. Even still, it's best to just remove this line completely and avoid trying to render the form partial twice.